docs/install/installing-istio.md

```diff
@@ -261,6 +261,24 @@ kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_
 **Note:** This method is only for development purposes. The production readiness of the above 
 installation method is not ensured. For a production-ready installation, see the `helm` installation method above.
 
+If you follow either of the above steps, your service and deployment for the local gateway are both named `cluster-local-gateway`,
+and you do not need to update gateway configmap `config-istio` under `knative-serving` namespace, because Knative Serving
```

Comment:
New sentence for readability;
"You do not need to update the `config-istio` configmap because Knative Serving can use the local gateway `cluster-local-gateway` by default."